src/AWS.Deploy.CLI/AWS.Deploy.CLI.csproj

```diff
@@ -21,15 +21,15 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="AWSSDK.IdentityManagement" Version="3.7.100.15" />
-    <PackageReference Include="AWSSDK.CloudFormation" Version="3.7.101.9" />
-    <PackageReference Include="AWSSDK.SecurityToken" Version="3.7.100.14" />
-    <PackageReference Include="AWSSDK.SSO" Version="3.7.100.14" />
-    <PackageReference Include="AWSSDK.SSOOIDC" Version="3.7.100.14" />
-    <PackageReference Include="Swashbuckle.AspNetCore.Annotations" Version="6.1.2" />
-    <PackageReference Include="Swashbuckle.AspNetCore.Swagger" Version="6.1.2" />
+    <PackageReference Include="AWSSDK.IdentityManagement" Version="3.7.200.42" />
```

Comment:
Could you update https://github.com/aws/aws-dotnet-deploy/blob/main/THIRD_PARTY_LICENSES and the internal tracker as well for all the version bumps?

Reply:
I went over all the dependencies and updated them. Also removed any dependency not referenced explicitly in the project.